readinglist/authentication.py

```diff
@@ -8,7 +8,10 @@
 
 
 def check_credentials(username, password, request):
-    """Basic auth backdoor. Used for testing (unit and loads)
+    """Basic auth backdoor.
+
+    This is mocked in the (unit and load) tests, and kept separate from the
+    authentication policies to ease the mocking.
```

Comment:
Hum, not really. Actually, it is not used in unit tests anymore. Just in loads tests. It is supposed to ease client development to bypass fxa integration...